### PR TITLE
Resolve media indicator versioning

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves #NUMBER

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-media-indicator/CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
-| 5.0.4 | [PR#3925](https://github.com/bbc/psammead/pull/3925) Talos - Bump Dependencies - @bbc/psammead-styles, @bbc/psammead-assets |
+| 6.0.1 | [PR#?](https://github.com/bbc/psammead/pull/?) Resolve versioning mismatch |
```
Comment (simonsinclair) [inline packages/components/psammead-media-indicator/CHANGELOG.md]:
```suggestion
| 6.0.1 | [PR#3929](https://github.com/bbc/psammead/pull/3929) Resolve versioning mismatch |
```

File: packages/components/psammead-media-indicator/CHANGELOG.md [hunk 2]
```diff
@@ -3,8 +3,9 @@
...
 | 6.0.0 | [PR#3898](https://github.com/bbc/psammead/pull/3898) Migrate to Emotion |
+| 5.0.4 | [PR#3925](https://github.com/bbc/psammead/pull/3925) Talos - Bump Dependencies - @bbc/psammead-styles, @bbc/psammead-assets |
```
Comment (simonsinclair) [inline packages/components/psammead-media-indicator/CHANGELOG.md:8]: Does this have the right PR linked?

**Review by RichardPK — APPROVED** [review]

File: packages/components/psammead-media-indicator/CHANGELOG.md (hunk 2 above)
Comment (RichardPK) [reply packages/components/psammead-media-indicator/CHANGELOG.md:8]: Aye it does SImon - this is the change within the linked PR https://github.com/bbc/psammead/pull/3925/files#diff-76112a7dcb40087e6179f84ace2ced7a29f199e854891e616166a66173a486faR6

Comment (RichardPK) [reply packages/components/psammead-media-indicator/CHANGELOG.md:8]: Eugh don't think that link takes you to the highlighted section - see the v5.0.4 bump within `packages/components/psammead-media-indicator/CHANGELOG.md`

**Review by simonsinclair — APPROVED** [review]

**Pull request merged**